Alyssa_McKee/Fundamentals/basic13.rb

```diff
@@ -0,0 +1,105 @@
+#I wrote multiple different methods, and uncommented the one I thought was the best
+
+puts "printing 1-255"
+(1..255).each{|i| puts i }
+
+#-------------------------------------------
+puts "\nprint odd nums between 1 and 255"
+
+(1..255).each{|i| puts i if i.odd?}
+#(1..255).step(2).each{|i| puts i} #also works
```

Comment:
Like you say, both of these work. The commented out one is pretty cool, since it reduce the amount of time taken, but a pretty big draw back considering it would have to start with an odd number